src/eu/packages/eu-component-button/docs/cta.md

```diff
@@ -14,7 +14,7 @@ npm install --save @ecl-twig/eu-component-button
 - "variant" (string) (default: 'primary'): variant of button (can be 'primary', 'secondary', 'call', 'ghost', 'search')
 - "type" (string) (default: 'submit'): can be the same type as HTML button - 'submit', 'reset', 'button'
 - "disabled" (bool) (default: false): define if button should be disabled (HTML disabled attribute)
-- "icon_position" [optional] (string) (default: after): Define icon position, can be 'before' or 'after'
+- "icon_position" [optional](string) (default: after): Define icon position, can be 'before' or 'after'
```

Comment:
Is it prettier that removed the space?

Shouldn't we alter the syntax in order to prevent making links? `[optional](string)` -> `(string, optional)`